docs/_docs/config.md

```diff
@@ -153,3 +153,24 @@ dynamoose.AWS = AWSXRay.captureAWS(require('aws-sdk'));
 ### dynamoose.setDocumentClient(documentClient)
 
 Sets the document client for DynamoDB. This can be used to integrate with Amazon Web Services DAX.
+
+### dynamoose.transaction(items[, options][, cb])
+
+Allows you to run DynamoDB transactions. Accepts an array as the `items` parameter. Every item in the `items` array must use the `Model.transaction` methods.
```

Comment:
Might be worth outlining that a regular, non-dynamoose query can also be passed into this method, as it simply accepts the request to send to Dynamo.

Reply:
@jutaz Sadly this is not actually the case. The custom `Model.transaction` methods attach the model itself to the object being passed in. This is so that `dynamoose.transaction` can correctly parse responses from DynamoDB. I THINK Dynamoose stores all the models somewhere tho. So maybe at some point we could change it to rely on that system, instead of needing to pass the model in. If we change that up at some point, then yes, I agree we should add that to the documentation.

Reply:
IIRC there's a place somewhere in Dynamoose in which we can get the Model reference by a table name, and that might be the key to make this work

Reply:
Nope, I think this is something that can be handled as a follow-up. Issue would be nice for this, I'd say 👍

Reply:
See #476